.editorconfig

```diff
@@ -0,0 +1,126 @@
+############################### 
+# Core EditorConfig Options   # 
+############################### 
+root = true
+[*.{cs,csx,vb,vbx,md}] 
+indent_style = space 
+indent_size = 4 
+insert_final_newline = true 
+charset = utf-8
+end_of_line = lf
```

Comment:
can this not be native eol instead?   otherwise this is causes warnings when switching from mac and windows

Reply:
@vis2k said to set it to LF, consistency is better than everything else